dashboard/src/components/nominations/NominationInformation.svelte

```diff
@@ -35,8 +35,8 @@
       { key: "nominee", value: "Nominee" },
       { key: "category", value: "Category" },
       { key: "nominator", value: "Nominated By" },
-      { key: "date", value: "Date", display: (date) => new Date(date).toLocaleString(),
-      sort: (a, b) => new Date(a) - new Date(b),},
```

Comment:
we should probably keep this in there as this allows us to sort the table by data automatically  when viewing the nominations page or we could also do this on the backend by adding and `ORDER BY` to the query